etc/picongpu/hemera-hzdr/gpu.tpl

```diff
@@ -41,7 +42,8 @@
 
 
 ## calculations will be performed by tbg ##
-.TBG_queue="gpu"
+.TBG_queue=${TBG_partition:-"gpu"}
+.TBG_account=`if [ $TBG_partition == "gpu_low" ] ; then echo "low"; else echo "default"; fi`
```

Comment:
Might be problematic.

Reply:
This is not problematic. The apperent higher priority originated from the fact that `gpu_low` encompass more nodes than `gpu` and thus could schedule my `low` job faster than my `default` one.